### DISCUSSION TITLE
how to set the gitlens proxy

### DISCUSSION THREAD

**Discussion by zhangliukun** [gitkraken/vscode-gitlens#1475] [Q&A]
It works fine when I pull code using the terminal. However, I use the vscode to pull code ,It reports the request error 403. It caused by the http_proxy. How could I change the http_proxy in the gitLens

**Comment by cyberluke** [comment]
same question

**Comment by cyberluke** [comment]
settings.js - what should be here? It is {} and not ""

    "gitlens.proxy": {
        
    },
